composer.json

```diff
@@ -12,7 +12,8 @@
   ],
   "require":      {
     "php":  ">=5.3.0",
-    "transfer/transfer": "~1.0"
+    "transfer/transfer": "~1.0",
+    "symfony/yaml": "2.7.5"
```

Comment:
This line should be moved to "require-dev" and "suggest" blocks. If you synchronize your branch with the 1.0 branch you'll see an example of how to do this. The reason behind this is that we don't want to enforce installation of libraries that might not be used by the end-user.